src/custom-element.js

```diff
@@ -297,7 +299,7 @@ function createBaseCustomElementClass(win) {
 
     /** @return {!Promise} */
     whenUpgraded() {
-      return this.signals_.whenSignal(CommonSignals.UPGRADED);
+      return this.signals_.whenSignal(CommonSignals.UPGRADED).then(EMPTY_FUNC);
```

Comment:
Why do we need the empty func?

Reply:
The `whenSignal` interface actually returns a `Promise<number>` :(

Reply:
Is that actually an issue anywhere? This makes all of these functions allocate 2 promises (and 2 loop ticks) instead of 1.

Reply:
Ok. Sure. Will remove. Just noticed and thought this was a bit unclean.